app/src/main/java/com/example/myapplication/FirstFragment.java

```diff
@@ -43,8 +43,8 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
                 binding.cont.setOnClickListener(v -> {
```

Comment:
The `onViewCreated` method is almost 100-lines long now. Consider moving some of the interfaces outside of it to make it more readable